fdbclient/DatabaseConfiguration.cpp

```diff
@@ -26,6 +26,14 @@
 #include "flow/genericactors.actor.h"
 #include "flow/UnitTest.h"
 
+StorageEngineParamsFactory redwoodFactory(
+    KeyValueStoreType::SSD_REDWOOD_V1,
+    { { "default_page_size", { StorageEngineParamSet::CHANGETYPE::NEEDREPLACEMENT, "8192" } },
+      { "kvstore_range_prefetch", { StorageEngineParamSet::CHANGETYPE::RUNTIME, "true" } },
+      { "metrics_interval", { StorageEngineParamSet::CHANGETYPE::NEEDREBOOT, "5.0" } },
```

Comment:
Yeah, the meeting was too long we first thought not need this but finally ended up this way.

Reply:
`default_page_size` should just be `page_size`.  The knob was labeled default because the knob does not represent a specific instance, just what to use for creation of new instances.

Reply:
This has been changed to `page_size`